crates/polars-ops/src/series/ops/is_between.rs

```diff
@@ -0,0 +1,34 @@
+use std::ops::BitAnd;
+
+use polars_core::prelude::*;
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq, Default)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub enum ClosedInterval {
```

Comment:
Too bad we can't reuse the existing `ClosedWindow` enum as it's in the `polars-time` crate which is not available here. But I guess it's fine this way.